samples/unity-of-bugs/Assets/Editor/Builder.cs

```diff
@@ -60,6 +60,7 @@ public static void BuildIl2CPPPlayer(BuildTarget buildTarget)
 
     public static void BuildWindowsIl2CPPPlayer() => BuildIl2CPPPlayer(BuildTarget.StandaloneWindows64);
     public static void BuildMacIl2CPPPlayer() => BuildIl2CPPPlayer(BuildTarget.StandaloneOSX);
+    public static void BuildLinuxIl2CPPPlayer() => BuildIl2CPPPlayer(BuildTarget.StandaloneLinux64);
```

Comment:
this ended up unused but I guess we can keep it for now. Later we can fix the smoke test on linux (currently times out)